homeworks/vladyslav-shevanov/Homework8/script.js

```diff
@@ -0,0 +1,29 @@
+function toggleClass(elem,className){
+    if (elem.className.indexOf(className) !== -1){
+        elem.className = elem.className.replace(className,'');
+    }
+    else{
+        elem.className = elem.className.replace(/\s+/g,' ') + 	' ' + className;
+    }
+    return elem;
+}
+
+function toggleMenuDisplay(e){
+    const dropdown = e.currentTarget.parentNode;
+    const menu = dropdown.querySelector('.dropdown_menu');
+    toggleClass(menu,'dropdown_hide');
+}
+
+function handleOptionSelected(e){
+    toggleClass(e.target.parentNode, 'dropdown_hide');
```

Comment:
```suggestion
    e.target.parentNode.classList.toggle("dropdown_hide");
```

Тоді можна видалити взагалі функцію 'toggleClass'